docs/getting-started.md

```diff
@@ -176,22 +176,29 @@ Follow the [installation instructions for your Linux distribution](https://nodej
 
 <block class='native windows android' />
 
-<h3>Node, Python2, JDK</h3>
+<h3>Node, Python, JDK</h3>
```

Comment:
Changing `"Python2"` to `"Python"` will results in version 3 installation. Was it the expected change?

Reply:
My bad. The windows build tools  uses Python 2.7 as a dependecy https://www.npmjs.com/package/windows-build-tools , but some other tools used by node have compatibility with Python v2.7, v3.5, v3.6, v3.7, or v3.8 https://github.com/nodejs/node-gyp#on-windows
This is the same problem in the changes from jdk8 to jdk14, I'll fix that by reverting the edit. 
Should I add information about tools compatibility with newer versions of Python and JDK?

Reply:
> Should I add information about tools compatibility with newer versions of Python and JDK?

I'm not sure if they will work (just with the warnings) or the installation procedure fails. I'm not that familiar with the backwards compatibility of Python and JDK. So, if you can test and confirm the behavior and based on that add note about compatibility it would be awesome.

Reply:
I've successfully built/compiled an React Native application using Python 3.8.4 and JDK 14 (I was using Python 2.7 with JDK14).
I'm using Windows 10 and the app was built using Node v10.16.3
I've added the note 'Some build tools have compability with some Python3 versions.', should we keep it?

Reply:
> I've added the note 'Some build tools have compability with some Python3 versions.', should we keep it?

Hmm, I think it's too general and providing the details will take too much space, so please remove the sentence. 

Rest of the changes LGTM, thank you!